docs/Speech and Audio/Setting Up Voice Commands/index.md

```diff
@@ -53,5 +53,38 @@ sdlManager.getScreenManager().setVoiceCommands([voiceCommand]);
 ```
 !@
 
+## Deleting Voice Commands
+To delete previously set voice commands, you just have to set an empty array or `nil` to the `voiceCommands` array on the screen manager.
+
+@![iOS]
+##### Objective-C
+```objc
+sdlManager.screenManager.voiceCommands = [];
+sdlManager.screenManager.voiceCommands = nil;
```

Comment:
```suggestion
```

Reply:
I was wrong. You can't set `nil` for `voiceCommands` and an error will be thrown at compile time